browser/css/notebookbar.css

```diff
@@ -633,5 +633,62 @@ button.ui-tab.notebookbar {
 }
 
 [class*='as-pptx-submenu-icon'] {
-	background: url('images/lc_downloadas-pptx.svg') center;
```

Comment:
Was this intentional? It changes the light icon to dark when using light theme.

Reply:
no, thank's for the review. will be fixed.